tests/Application/config/sylius/2.0/packages/security.yaml

```diff
@@ -0,0 +1,120 @@
+security:
```

Comment:
I do not get, why we have doubled configs, i.e. `tests/Application/config/packages/security.yaml` and the current one?